.add/lib/redux/model/admin/types.ts

```diff
@@ -0,0 +1,17 @@
+import { ProposedArtist } from "../types";
+
+export interface AdminState {
+    loading: boolean;
+    autocompletedArtists: ProposedArtist[];
+    error: string | undefined;
+    djs: DJ[];
+};
+
+export interface DJ {
+    userName: string;
+    realName: string;
+    djName: string;
+    isAdmin: boolean;
```

Comment:
This is something that we might need to revisit later. It's possible that we'll have a variety of roles in the system and we'll need something more complex than a boolean. This is fine for now, though.